recipes/dd4hep/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "dd4hep" %}
+{% set version = "01-13" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version.split('-')|map('int')|join('.') }}"
+
+source:
+  url: https://github.com/AIDASoft/DD4hep/archive/v{{ version }}.zip
+  sha256: 945a04cd04e9af23cdd4ba29a41bbff0fc388a66e5b640eb3f633596f130573d
+  folder: source
+
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    - cmake
+    - ninja
```

Is this actually using ninja instead of `make`?
```suggestion
    - make
```